console/console-init/ui/src/modules/device/components/DeviceListAlert/DeviceListAlert.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { Alert } from "@patternfly/react-core";
+
+export interface IDeviceListAlertProps {
```

Comment:
always try to use interface of  PF component to avoid define same properties again in our components.
Here you can extend PF Alert interface AlertProps and removed properties form IDeviceListAlertProps interface whic are available in PF interface. title and variant properties are available in PF interface AlertProps. so you can remove.
export interface IDeviceListAlertProps extends AlertProps {
}